README.md

```diff
@@ -42,7 +42,7 @@ Load `@octokit/graphql` directly from [cdn.pika.dev](https://cdn.pika.dev)
 Node
 </th><td>
 
-Install with <code>npm install @octokit/graphql</code>
+Install with <code>npm install --save @octokit/graphql</code>
```

Comment:
I think the `--save` is obsolete for all npm versions that are distributed with currently maintained Node versions?

Reply:
Learned something new today.